### PR TITLE
feat: New parser for systemd_perms

### PR DESCRIPTION
**Pull request by TZ3070**
Signed-off-by: Xinting Li <xintli@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

*Add your description here*

### CODE REVIEW

**Comment by TZ3070** [comment]
@xiangce Please have a review.

**Review by xiangce — CHANGES_REQUESTED** [review]
@TZ3070 - These two parsers are in fact `Ls` like parsers, would you please rename them with a prefix `Ls` like others, e.g. 
https://github.com/RedHatInsights/insights-core/blob/cc2e26225b1db85eb11e3b160cb12b81d1b3796c/insights/parsers/ls_sys_firmware.py#L45-L46

File: insights/parsers/ls_systemd.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
+"""
+Systemd File Permissions parsers
+================================
+
+Parsers included in this module are:
+
+SystemdPerms - command ``/bin/ls -lanRL /etc/systemd``
```
Comment (xiangce) [inline insights/parsers/ls_systemd.py:7]: This should be modified too.

File: insights/parsers/ls_systemd.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+--------------------------------------------------------------
+
+UsrLibSystemdPerms - command ``/bin/ls -lanRL /usr/lib/systemd``
```
Comment (xiangce) [inline insights/parsers/ls_systemd.py:10]: The same comment as above

File: insights/parsers/ls_systemd.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+
+"""
+from insights.core import CommandParser, FileListing
+from insights.core.plugins import parser
+from insights.specs import Specs
+
+
+@parser(Specs.ls_etc_systemd)
+class LsEtcSystemdPermsParser(CommandParser, FileListing):
```
Comment (xiangce) [inline insights/parsers/ls_systemd.py:21]: How above remove the part of `PermsParser`?

File: insights/parsers/ls_systemd.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    >>> etc_systemd_dicts['coredump.conf']['perms']
+    'rw-r--r--'
+    >>> etc_systemd_dicts['user']['perms']
+    'rwxr-xr-x'
+    """
+    pass
+
+
+@parser(Specs.ls_usr_lib_systemd)
+class LsUsrLibSystemdPermsParser(CommandParser, FileListing):
```
Comment (xiangce) [inline insights/parsers/ls_systemd.py:69]: How above remove the part of `PermsParser`?

**Review by xiangce — CHANGES_REQUESTED** [review]
@TZ3070 - Please check my new comments above

And 1) since this is a new parser file, please add doc entry for it in the `docs/shared_parsers_catalog/` dir, so that its online document can be generated automatically. 2), for newly added specs parsers, please raise an MR to the insights-core-assets repo at first to make sure they can be added to the uploader.json.

File: insights/specs/default.py [hunk 5]
```diff
@@ -371,6 +371,7 @@ def httpd_cmd(broker):
         "/etc/pki/ovirt-vmconsole", "/etc/nova/migration", "/etc/sysconfig",
         "/etc/cloud/cloud.cfg.d", "/etc/rc.d/init.d"])
     ls_etc = simple_command("/bin/ls -lan {0}".format(' '.join(etc_and_sub_dirs)), keep_rc=True)
+    ls_etc_systemd = simple_command("/bin/ls -lanRL /etc/systemd")
```
Comment (xiangce) [inline insights/specs/default.py:374]: You also need to add the `simple_command` spec to the `insights_archive.py` as a `simple_file` to make sure the `core` could find it from the insights-archive successfully. 

Please use the `mangle` command provide by the insights-core to get the `simple_file` of this `simple_command`

File: insights/specs/default.py [hunk 6]
```diff
@@ -382,6 +383,7 @@ def httpd_cmd(broker):
     ls_tmp = simple_command("/bin/ls -la /tmp")
     ls_usr_bin = simple_command("/bin/ls -lan /usr/bin")
     ls_usr_lib64 = simple_command("/bin/ls -lan /usr/lib64")
+    ls_usr_lib_systemd = simple_command("/bin/ls -lanRL /usr/lib/systemd")
```
Comment (xiangce) [inline insights/specs/default.py:386]: Same as above

**Review by xiangce — APPROVED** [review]

**Review by xiangce — APPROVED** [review]
Approved pending the approval of Specs.

**Pull request merged**